selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/client/keycloak/KeycloakAdminConsoleClient.java

```diff
@@ -212,14 +167,13 @@ private String extractUserId(String response) {
   private void delete(String userId, String username) throws IOException {
     String commandToDeleteUser =
         format(
-            "docker exec -i %s sh -c 'keycloak/bin/kcadm.sh delete users/%s -r che -s username=%s --no-config --server http://localhost:8080/auth --user %s --password %s --realm master 2>&1'",
-            keycloakContainerId,
+            "delete users/%s -r che -s username=%s --no-config --server http://localhost:8080/auth --user %s --password %s --realm master 2>&1",
```

Comment:
What about a case when "CHE_PORT" variable is different from default "8080"?

Reply:
Here we are using different port - for keycloak server from within keycloak container.